ui/src/ResponsiveAppBar.js

```diff
@@ -184,7 +188,14 @@ function ResponsiveAppBar({ user }) {
           <Box sx={{ flexGrow: 0 }}>
             <Tooltip title="Open settings">
```

Comment:
My main objective was to change this. We don't have settings. Maybe something like:

```suggestion
            <Tooltip title="{`${user.given_name} ${user.family_name}`}">
```

or

```suggestion
            <Tooltip title="User details">
```

(Untested.)

Reply:
Yes totally, that's a change I had that I didn't commit, I suggest: "Navigation" since we're able to navigate to different pages or logout, but if you prefer I can also use "User details"